src/deepsparse/server/openai_server.py

```diff
@@ -175,6 +212,107 @@ async def abort_request() -> None:
             )
             return response
 
+        @app.post(
+            "/v1/completions",
+            tags=["model", "inference"],
+            response_model=CompletionResponse,
+        )
+        async def create_completion(raw_request: Request):
+            request = CompletionRequest(**await raw_request.json())
+            _LOGGER.info(f"Received completion request: {request}")
```

Comment:
Server could receive a multitude of requests, would logging the request each time generate a sea of logs?

Reply:
yes. will remove